tests/crds/cloudscale.crossplane.io_objectsusers.yaml

```diff
@@ -0,0 +1,314 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
+  creationTimestamp: null
+  name: objectsusers.cloudscale.crossplane.io
```

Comment:
Please add make automation that downloads the CRD from GitHub releases instead of manually maintaining this.
You can extract the version number used from the package, e.g. `yq e '.parameters."pkg.appcat.provider.cloudscale".images.provider-cloudscale.tag' packages/provider/cloudscale.yml` and probably it makes sense to exclude this file from git completely.

Reply:
Added that.